README.md

```diff
@@ -1,3 +1,10 @@
 This repository houses various examples linked to from our [documentation](https://xata.io/docs). PRs are welcome.
 
+* [Astro](https://github.com/xataio/examples/tree/main/apps/getting-started-astro)
+* [Next.js](https://github.com/xataio/examples/tree/main/apps/getting-started-nextjs)
+* [Nuxt](https://github.com/xataio/examples/tree/main/apps/getting-started-nuxt)
+* [Remix](https://github.com/xataio/examples/tree/main/apps/getting-started-remix)
+* [SolidStart](https://github.com/xataio/examples/tree/main/apps/getting-started-solidstart)
+* [SvelteKit](https://github.com/xataio/examples/tree/main/apps/getting-started-sveltekit)
```

Comment:
If we're gonna do this, we might as well list the other examples?

Reply:
addressed the full list in https://github.com/xataio/examples/pull/600/commits/a6da38ab4f1c5aedff1dd5c1a253a3c3642fd0e0